app/src/Model/Table/Article.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+
+// src/Model/Entity/Article.php
+namespace App\Model\Entity;
```

Comment:
I noticed this while reviewing PR #3.

Refer https://book.cakephp.org/4/en/orm/entities.html#creating-entity-classes

> By convention entity classes live in src/Model/Entity/

Please kindly move to the correct directory for entity class.

Reply:
Was moved in https://github.com/combimauri/tutorial-cakephp/pull/5/files#diff-f406269e246cb6dca1c08aed02d3a196bbaf2cef8051d23430c4d3a3d92eb628

Reply:
Yes, I noticed this mistake later in the process, it was causing issues when implementing policies, that's why I fixed it in the Athorization PR